artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/TokenReview.java

```diff
@@ -0,0 +1,69 @@
+
+import org.apache.activemq.artemis.spi.core.security.jaas.kubernetes.client.KubernetesClient;
+import org.apache.activemq.artemis.spi.core.security.jaas.kubernetes.client.KubernetesClientImpl;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "token-review", description = "Perform a kubernetes token review")
+public class TokenReview extends InputAbstract {
+
+   @Override
+   public Object execute(ActionContext context) throws Exception {
+      super.execute(context);
+
+      context.out.println();
+      String kubeHost = input("--kube-host", "What is the cluster host?", "api.crc.testing");
+      String kubeport = input("--kube-port", "What is the cluster port?", "6443");
+      String tokenpath = input("--token-path", "What is the token path?", "/var/run/secrets/kubernetes.io/serviceaccount/token");
+      String capath = input("--ca-path", "What is the ca path?", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt");
+      String prevKSH = System.getProperty("KUBERNETES_SERVICE_HOST", kubeHost);
+      String prevKSP = System.getProperty("KUBERNETES_SERVICE_PORT", kubeport);
+      String prevTP = System.getProperty("KUBERNETES_TOKEN_PATH", tokenpath);
+      String prevCP = System.getProperty("KUBERNETES_CA_PATH", capath);
+      System.setProperty("KUBERNETES_SERVICE_HOST", kubeHost);
+      System.setProperty("KUBERNETES_SERVICE_PORT", kubeport);
+      System.setProperty("KUBERNETES_TOKEN_PATH", tokenpath);
+      System.setProperty("KUBERNETES_CA_PATH", capath);
+      String token = input("--token", "What is the token?", "sha256~Cg_LIHObUgJHvF_Ziv9k9eHZQ3_UUNihRD9vA96hQKU");
+      try {
+         KubernetesClient client = new KubernetesClientImpl();
+         org.apache.activemq.artemis.spi.core.security.jaas.kubernetes.model.TokenReview tr = client.getTokenReview(token);
+         context.out.println("*******************************************************************************************************************************");
+         context.out.println("* Performing a token review on:");
+         context.out.println("* host:: " + kubeHost);
+         context.out.println("* port:: " + kubeport);
+         context.out.println("* tokenpath:: " + tokenpath);
+         context.out.println("* capath:: " + capath);
+         context.out.println("*");
+         context.out.println("* Result:");
+         context.out.println("* token:: " + token);
+         context.out.println("* username:: " + tr.getUsername());
+         context.out.println("*******************************************************************************************************************************");
+         context.out.println();
+      } finally {
+         System.setProperty("KUBERNETES_SERVICE_HOST", prevKSH);
+         System.setProperty("KUBERNETES_SERVICE_PORT", prevKSP);
+         System.setProperty("KUBERNETES_TOKEN_PATH", prevTP);
+         System.setProperty("KUBERNETES_CA_PATH", prevCP);
+      }
```

Comment:
I'd be inclined to do something so that specific option values could neatly be passed in rather than resorting to the 'get + set + run + reset'. E.g pass in an 'environment' map of options, kind of like the way JNDI InitialContext stuff works. If the value is specified that way, use it, otherwise pull the value the way it currently would have been.